src/sigchain/sigchain.iced

```diff
@@ -76,6 +76,19 @@ bad_whitespace_sig_ids = {
   "db9a0afaab297048be0d44ffd6d89a3eb6a003256426d7fd87a60ab59880f8160f" : true
 }
 
+# Sometimes a bug in our software leads to signatures that we will never be
+# able to validate. (Sorry Fred!) We can't just delete these, because that
```

Comment:
BTW, these links weren't buggy.  We just changed the format shortly thereafter and didn't want to keep backwards-compatible code for something that never saw the light of day... Erm, nearly didn't see the light of day...